### PR TITLE
Some cleanup

### PR DESCRIPTION
**Pull request by arjunguha**

### CODE REVIEW

File: src/callcc/callcc.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import * as label from './label';
+import * as jumper from './jumper';
+import * as declVars from './declVars';
+import * as nameExprs from './nameExprs';
+import { transform, letExpression } from '../common/helpers';
+import { NodePath, Visitor } from 'babel-traverse';
+import * as babylon from 'babylon';
+import * as t from 'babel-types';
+import * as babel from 'babel-core';
+import * as fs from 'fs';
```
Comment (baxtersa) [inline src/callcc/callcc.ts]: Does `fs` need to be imported? I don't see it being used.

Comment (arjunguha) [inline src/callcc/callcc.ts]: good points thanks

File: src/callcc/boxAssignables.ts [hunk 2]
```diff
@@ -13,7 +27,23 @@ function boxVars(path: NodePath<t.Node>, vars: string[]) {
...
+      }
+    },
     Identifier(path: NodePath<t.Identifier>) {
+      const parent = path.parent;
+      if (parent.type === "FunctionExpression" ||
+          parent.type === "FunctionDeclaration") {
+        path.skip();
+        return;
+      }
+      if (parent.type === "MemberExpression" &&
```
Comment (baxtersa) [inline src/callcc/boxAssignables.ts]: I believe you can avoid an `any` cast here by doing
`if (t.isMemberExpression(parent) && parent.property === path.node) { ...`

`t.isMemberExpression(node)` has return type `node is MemberExpression`, typescript seems to be able to refine on this.

**Comment by baxtersa** [comment]
Oh, I just realized a bunch of tests fail. I'll work on them tomorrow.

**Comment by rachitnigam** [comment]
@arjunguha, you forgot to answer the two comments I had about the PR. Can you answer them?

**Pull request merged**